packages/eventive/src/EventivePlugin.ts

```diff
@@ -4,5 +4,10 @@ export type EventivePlugin<
   DomainEvent extends BaseDomainEvent<string, {}>,
   State extends {}
 > = {
+  beforeCommit?(args: {
+    event: DomainEvent;
+    entity: BaseEntity<State>;
+    abortCommit: () => void;
```

Comment:
```suggestion
    abort: () => void;
```

I think `abort` is more consistent naming